.github/workflows/maestro-local.yml

```diff
@@ -62,8 +60,9 @@ jobs:
   maestro-cloud:
     name: Maestro test suite
     runs-on: ubuntu-latest
-    needs: build-apk
-    if: github.event_name == 'workflow_dispatch' || github.event.label.name == 'Run-Maestro'
+    needs: [build-apk]
```

Comment:
...but I am confused with this needs statement.

Does this means that this `yml` is also able to build an APK on-demand (using `workflow_dispatch`)?